src/app/static/src/app/components/figures/dynamicTable.vue

```diff
@@ -41,7 +41,10 @@
                     return "n/a";
                 }
                 if (typeof value == "string") {
-                    return value;
+                    if (isNaN(parseFloat(value))) {
```

Comment:
These additions are because now in some series we have both "n/a" and numerical values, so they get serialised as strings and need to be explicitly parsed here.